services/ui-src/src/cmit.ts

```diff
@@ -0,0 +1,17 @@
+import { CMIT, DeliverySystem, DataSource, MeasureSpecification } from "types";
```

Comment:
Brendan said it was fine for me to move it here for now if it keeps things moving. We're still figuring out how we want the report to access this data and where